.gitignore

```diff
@@ -1,4 +1,6 @@
 /vendor/
 composer.lock
 .php-cs-fixer.cache
-.phpunit.result.cache
```

Comment:
Can you revert this btw? No need to have those specifics in this project, otherwise we'll end up with an endlessly long gitignore file with all kinds of non-project related stuff :)